### DISCUSSION TITLE
streams-generator-i2s -> disturbing noise with ESP32 and PCM5102A

### DISCUSSION THREAD

**Discussion by tobo-123** [pschatzmann/arduino-audio-tools#1764] [Q&A] [answered]
Hello,

I am a total beginner with this library and started with the basic example “streams-generator-i2s”. I am using an ESP32 Wroom DevKit V1 and a PCM5102A external DAC with headphones. I only want to play one tone. I have set the core debug level in the Arduino IDE to “Warn”, I guess that is meant by "Please note the log level should be set " in the documentation.

When I start the program on the ESP32, I get the tone, but with a lot of annoying noises and clicks. I have found that if I comment out the AudioLogger, everything works fine.

What am I doing wrong here?

Regards,
tobo

**Comment by pschatzmann** [comment, marked as answer]
No, you are supposed to set the AudioTools log level to warning

    AudioLogger::instance().begin(Serial, AudioLogger::Warning);

Which is actually the same as commenting it out...

**Reply by tobo-123** [reply]
Thanks a lot for the fast reply :)
